### PR TITLE
fix: create platform handler isPlatform flag

### PR DESCRIPTION
**Pull request by supalarry**
We received "isPlatform" property but I forgot to add it to data used to create team.

### CODE REVIEW

**Review by ThyMinimalDev — DISMISSED** [review]

**Review by keithwillcode — DISMISSED** [review]

**Review by zomars — DISMISSED** [review]

**Review by zomars — DISMISSED** [review]

**Review by zomars — APPROVED** [review]

File: packages/lib/server/repository/organization.ts [hunk 1]
```diff
@@ -45,13 +45,13 @@ export class OrganizationRepository {
...
             isOrganizationConfigured: orgData.isOrganizationConfigured,
             orgAutoAcceptEmail: orgData.autoAcceptEmail,
-            isPlatform: orgData.isPlatform,
           },
         },
         metadata: {
           ...(IS_TEAM_BILLING_ENABLED ? { requestedSlug: orgData.slug } : {}),
           orgSeats: orgData.seats,
           orgPricePerSeat: orgData.pricePerSeat,
+          isPlatform: orgData.isPlatform,
```
Comment (keithwillcode) [inline packages/lib/server/repository/organization.ts:54]: @zomars Why was this moved to metadata here? There is a standalone column `isPlatform` on the team table that the platform team has intended to use.
